ui/app/AppLayouts/Onboarding2/pages/LoginScreen.qml

```diff
@@ -80,7 +80,7 @@ OnboardingPage {
     signal onboardingLoginFlowRequested()
     signal unblockWithSeedphraseRequested()
     signal unblockWithPukRequested()
-    signal lostKeycard()
+    signal lostKeycard(string keyUid)
```

Comment:
Ideally, for consistency the name of signal should be `lostKeycardFlowRequested`